source/_components/sensor.buienradar.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: "buienradar"
+description: "Instructions how to integrate buienradar.nl sensor within Home Assistant."
+date: 2017-05-15 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: 
+ha_category: Weather
+ha_release: 0.44
+---
+
+
+The `buienradar` platform uses [buienradar.nl](http://buienradar.nl/) as an source for current meteorological data for your location. The 
+weather forecast is delivered by Buienradar, who provides a webservice that provides detailed weather information for users in The Netherlands.
+The relevant weatherstation used will be automatically selected based on the location specified in the HA config (or in the buienradar weather/sensor component).
```

Comment:
Avoid the use of HA and instead write out the full name of Home Assistant. Similarly config should be configuration.